projects/core/src/cms/facade/cms.service.ts

```diff
@@ -73,7 +73,7 @@ export class CmsService {
   getComponentData<T extends CmsComponent>(uid: string): Observable<T> {
     if (!this.components[uid]) {
       this.components[uid] = this.routingService.isNavigating().pipe(
-        withLatestFrom(
+        combineLatest(
```

Comment:
The `combineLatest` operator is deprecated in favour of `combineLatest` factory function. So please change:
`import { combineLatest } from 'rxjs/operators'` to `import { combineLatest } from 'rxjs'`.

And use it differently in the code:
```
this.components[uid] = combineLatest(
        this.routingService.isNavigating(),
        this.store.pipe(select(fromStore.componentStateSelectorFactory(uid)))
      ).pipe(
        tap( ....
...
```